backend/fms_core/template_importer/row_handlers/project_link_samples/project_link_samples.py

```diff
@@ -28,18 +32,110 @@ def process_row_inner(self, sample, project, action):
                               f"is named {sample_obj.name} not {sample['sample_name']}."
                 self.warnings['sample'].append(warning_msg)
 
-            # Check if link exists to ensure there's not a duplicated association
-            if action['name'] == ADD_ACTION:
+            # Perform an add project and/or study action
+            if action['name'] == ADD_PROJECT_ACTION:
                 # Create link object if no errors
                 link_created, self.errors['link'], self.warnings['link'] = create_link(sample=sample_obj,
                                                                                        project=project_obj)
 
-            # If the link doesn't exists we can't perform a remove action
-            elif action['name'] == REMOVE_ACTION:
-                # Remove link object if no errors
-                link_removed, self.errors['link'], self.warnings['link'] = remove_link(sample=sample_obj,
-                                                                                       project=project_obj)
+                # Queue sample to study if provided
+                if project['study_letter']:
+                    study_obj, self.errors['study'], self.warnings['study'] = get_study(project_obj, project['study_letter'])
+
+                    if study_obj:
+                        # To avoid empty step orders
+                        step_order = project['step_order'] if project['step_order'] else 1
```

Comment:
1 is not the default of all studies ... queue_sample_to_study_workflow would take care of it if you left it empty.